src/CustomItem.cs

```diff
@@ -51,31 +49,44 @@ public CustomItem(
 		// C# pls
 		if (soldOnlyAt == default(List<Shop>)) { soldOnlyAt = new List<Shop>(); }
 		Main.Log($"Instantiating {itemInfo.Name}");
+
+		// save item info for later use
+		//this.itemInfo = itemInfo;
 		
 		Name = itemInfo.Name;
 		Description = itemInfo.Description;
-		ItemPrefab = CreateItemPrefab(unityPrefab);
-		ProvidedPrefab = unityPrefab;
+		ItemPrefab = CreateItemPrefab(providedItemPrefab);
+		//this.providedItemPrefab = providedItemPrefab;
```

Comment:
Please remove the commented lines of code, if they're not needed anymore

Reply:
Removed (and another one)